Keas.Mvc/ClientApp/components/Documents/DocumentsContainer.tsx

```diff
@@ -15,36 +15,37 @@ interface IProps {
   person: IPerson;
 }
 
-export const DocumentsContainer = (props: IProps): JSX.Element => {
+const DocumentsContainer = (props: IProps): JSX.Element => {
```

Comment:
@srkirkland This is different from the other containers. but it looks like it works.

Reply:
@jSylvestre yea, this is "new React" aka React with hooks.